server.js

```diff
@@ -2,7 +2,8 @@ const services = [
   require('dogstack-agents/service'),
   require('./tasks/services/plans'),
   require('./tasks/services/works'),
-  require('./ordering/services/orders')
+  require('./ordering/services/orders'),
+  require('./agents/services/hooks/agents')
```

Comment:
my reasoning is that this is not actually a special case, this is just a normal service. my intent in changing services to be the feathers plugin pattern, rather than a feathers service, was to avoid creating special cases, as i realized these are super common and i found it to become an anti-pattern to try and figure out the differences, especially when they all blend together (this is not the only service with hooks, for example).